CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
```

Comment:
Your prettier seems off :p

Reply:
The `h3` is next to the `h2` because of that thing @klzns mentioned when git is detecting conflicts in this file 🤷‍♂